src/pages/docs/octopus-cloud/getting-started-with-cloud.mdx

```diff
@@ -23,78 +23,90 @@ After you create an Octopus account, you can create a new instance of Octopus Cl
 
 To create a new instance of Octopus Cloud, make sure you’re logged in to your Octopus account:
 
-1. Click **Create a new cloud instance**.
-2. Enter the URL you’d like to use to access your instance: `your-url.octopus.app` . If the URL isn’t available, enter a different URL.
+1. Click **New Cloud Instance**.
+2. Enter the URL you’d like to use to access your instance: `your-url.octopus.app`. If the URL isn’t available, enter a different URL.
 3. Select the region where you’d like to host your instance.
 4. Add your company name.
 5. Review the terms of our customer agreement, privacy policy, and acceptable usage policy.
-6. Click **Agree, deploy my Octopus**!
+6. Click **Start your trial**.
 
 You’ll be taken to the account provisioning screen. Your Octopus Cloud instance should be ready in a minute or so. You’ll receive an email when the instance is ready to use.
 
-When the instance is ready, you’ll see it (and any other instances you have access to) the next time you log in to your Octopus account at [octopus.com/signin](https://octopus.com/signin).
+## Managing Octopus Cloud subscriptions \{#managing-octopus-cloud-subscriptions}
 
-## Uploading packages to Octopus Cloud servers \{#uploading-packages-to-octopus-cloud-servers}
```

Comment:
This section was introduced in #375 over 5 years ago. I've taken it out because I don't think it belongs on a "Getting Started" page.

I think in general people understand how networks work, and this note is probably only useful to a customer migrating from self-hosted to Cloud. The context there would be if they were deploying from self-hosted to network-close targets (e.g. on-prem to on-prem, or self-hosted in Cloud to targets within the same Cloud region).

A better place for this might be `Migrating to Cloud`, I might add it back there when I get to that doc.